### ISSUE TITLE
wasm create process fails with an error on current feat-triple-masking-v2  branch

### ISSUE THREAD

**Issue by okovalov** [FindoraNetwork/platform#122]
### Problem Description
>Trying to create wasm on current develop branch, and the process fails and throws an error:

>Screenshots:

<img width="872" alt="CleanShot 2021-11-22 at 16 16 01@2x" src="https://user-images.githubusercontent.com/1706749/142936397-41ff6d28-4412-47d9-9a30-0d0beb840a00.png">

<img width="979" alt="CleanShot 2021-11-22 at 16 16 26@2x" src="https://user-images.githubusercontent.com/1706749/142936433-cc509fdb-aafe-4a02-99b9-903e745cc39a.png">

<img width="1036" alt="CleanShot 2021-11-22 at 16 16 45@2x" src="https://user-images.githubusercontent.com/1706749/142936454-b20758ce-a250-45b5-87ab-a48579667aab.png">

### Environmental information
>operating system:

![](https://user-images.githubusercontent.com/1706749/142912875-195bad6e-d9e0-472c-a26c-17974882487b.png)

>Code version:

<img width="965" alt="CleanShot 2021-11-22 at 16 17 51@2x" src="https://user-images.githubusercontent.com/1706749/142936584-6ff1ab6a-41e4-4b4d-8142-17fa758e671d.png">

**Note**: it might be related to https://github.com/FindoraNetwork/platform/issues/120 

### Steps to reproduce
<img width="785" alt="CleanShot 2021-11-22 at 16 18 33@2x" src="https://user-images.githubusercontent.com/1706749/142936672-ab2e953f-fa9c-4575-9305-f85c4cd90972.png">

**Comment by okovalov** [comment]
ok, @kingpinXD the problem is partially solved, meaning the wasm is being created but it is not working. 

unfortunately, there is no other way but merge develop into a feature branch. sdk build fails because there is a lot of evm and staking stuff has been fixed in develop

![image](https://user-images.githubusercontent.com/1706749/144108978-e95842ca-4649-4727-8af2-927a1167d174.png)

![image](https://user-images.githubusercontent.com/1706749/144109017-f3baa36a-204a-437f-9482-074ffaab17c3.png)

yes, I can disable that to fake proper build of the sdk, but then I wont be able to merge that feature branch of sdk into main, as it is going to use wasm with broken evm and staking..

sooner we do that easier that would be...

also, another issue - for some reason `get_anon_keys` has two different signatures in declaration and implementation. sdk is getting confused. I am trying to force it to use `AnonKeys`

![image](https://user-images.githubusercontent.com/1706749/144109146-bfad95fa-9d97-4d92-b6d0-f0d773b54713.png)

![image](https://user-images.githubusercontent.com/1706749/144109164-92a682c0-2185-4027-b232-24936cbb8604.png)

but the biggest problem is that the wasm which was created is not importable , even though I used a correct target
![image](https://user-images.githubusercontent.com/1706749/144108616-efb6f68c-9713-41b6-8aca-8a9b48f4efe9.png)


where is the wasm from the stable tag works is being imported just fine `"findora-wallet-wasm": "https://github.com/FindoraNetwork/wasm-js-bindings.git#testnet-evm-v0.2.2",`


I am assuming there is some difference between the libraries or the way the wasm is being compiled..

btw this  screenshow was from the run when I used `-t nodejs` but I have also tried all the targets and `make wasm` too. as far as I understand `make wasm` runs -t bundler because it is producing `wasm_bg.js` whereas it is not been created when you run `-t nodejs`

so it fails too (which is expected since the target is not correct)

![image](https://user-images.githubusercontent.com/1706749/144108770-8b087104-13e4-49d9-93c5-3361aa6400f6.png)

to summarize:
- I think it is a good start but we must merge develop at some point rather sooner than later (MEDIUM PRIORITY)
- we need to understand and fix the problem of different function signatures (LOW PRIORITY)
- we need to update make wasm to be able to build for different targets, currently it seems to be building for bundler (HIGH PRIORITY)
- we have to use the same command/library/parameters that is used in this tag `"https://github.com/FindoraNetwork/wasm-js-bindings.git#testnet-evm-v0.2.2"` to be able to create an importable wasm, otherwise we can't proceed at all (HIGHEST PRIORITY)

**Comment by harshadptl** [comment]
This branch is now deperecated. All the triple masking related changes are merged in develop where the wasm build succeeds
